docs/modules/plugins/partials/dynamic-variables-ui.adoc

```diff
@@ -57,3 +57,31 @@ ${source-code}
 ----
 Then `${source-code}` matches `.+Home.+`
 ----
+
+=== Browser window dimension
```

Comment:
it's not available for mobile native apps, but `dynamic-variables-ui.adoc` contains info about vars common for both web and native mobile apps